src/main/resources/data/computercraft/lua/rom/programs/edit.lua

```diff
@@ -47,6 +47,31 @@ else
     stringColour = colours.white
 end
 
+local runHeader = [[local current = term.current()
+local ok,err = load([===============]] .. [[==================[]]
+local runHandler = [[]===============]] .. [[==================], "@.temp", nil, _ENV)
+if ok then ok,err = pcall(ok,...) end
```

Comment:
Stupid nitpick, sorry, but can you add spaces after all the commas. It's consistent with the rest of the code - it's just the linter won't pick up on this.